examples/webgl_materials_texture3d_volume1.html

```diff
@@ -203,6 +203,7 @@
 			container.appendChild( renderer.domElement );
 
 			controls = new THREE.OrthographicTrackballControls( camera, renderer.domElement );
+			controls.target = new THREE.Vector3(64, 64, 128);
```

Comment:
```javascript
controls.target.set( 64, 64, 128 );
```

Reply:
@WestLangley thanks! Corrected. Still getting used to Three :)